### DISCUSSION TITLE
Any idea why I'm seeing this?

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#470] [Help]
I downloaded Malwarebytes to solve a problem with Microsoft Edge. When I went to run Freeplane, it came up with this.

![image](https://user-images.githubusercontent.com/8197883/175875299-67d7ade4-953a-461b-b166-53107d33edd4.png)

Malwarebytes quarantined Freeplane.exe. It will let me restore it if I deem things to be safe.

Are things safe? 

Any idea why I'm seeing this, or what MachineLearning/Anomalous.96% is?

~ Bal

**Comment by macmarrum** [comment]
It's worth to know that `freeplane.exe` uses the [launch4j](http://launch4j.sourceforge.net/) Java executable wrapper. The quarantine action you are seeing might be a false positive. This is what I found when I searched the forums of malwarebytes: https://forums.malwarebytes.com/topic/254200-malwarebytes-is-removing-my-program/?do=findComment&comment=1347947

If you are still concerned, you can use `freeplane.bat` instead of `freeplane.exe`

**Comment by wordmuse** [comment]
Thank you. I am tending to ignore this as a false positive. But I don't really get your comment about the freeplane.bat. I mean, I know what a .bat file is, but I don't see freeplane.bat in the freeplane folder on my computer. Is this a file you create? And wouldn't it call the .exe file? If so, how would it help?

Thanks for responding.

~ Bal

**Reply by macmarrum** [reply]
Yep, it looks like a false positive to me.

Freeplane is a Java application. This means that the exact command to start it is `java.exe --add-exports=java.desktop/sun.awt.shell=ALL-UNNAMED --add-exports=java.desktop/com.sun.java.swing.plaf.windows=ALL-UNNAMED -Xmx512m -Dorg.freeplane.userfpdir="%APPDATA%\\Freeplane" -jar freeplanelauncher.jar %*`

`freeplane.exe` is a utility that executes the command behind the scenes, doing some other stuff before, and makes sure that Windows Terminal window is not displayed (i.e. that java runs in the background). In contrast, `freeplane.bat` will create a Windows Terminal window, aka Command Prompt, in addition to the main Freeplane GUI.

As far as I know, `freeplane.bat` comes with each Freeplane installation. If you are missing `freeplane.bat`, you can download [freeplane_bin-1.10.3.zip](https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_bin-1.10.3.zip/download) and extract it from there. You can also create it (new file) in your `C:\Program Files\Freeplane`, adding the following content:
```
java ^
--add-exports=java.desktop/sun.awt.shell=ALL-UNNAMED ^
--add-exports=java.desktop/com.sun.java.swing.plaf.windows=ALL-UNNAMED ^
-Xmx512m -Dorg.freeplane.userfpdir="%APPDATA%\\Freeplane" ^
-jar freeplanelauncher.jar %*
```

**Reply by wordmuse** [reply]
got it - thanks again. :)
